datalad/utils.py

```diff
@@ -867,7 +943,7 @@ def ensure_unicode(s, encoding=None, confidence=None):
         return s.decode(encoding)
 
 
-def ensure_bool(s):
+def ensure_bool(s: str | bool) -> bool:
```

Comment:
I wonder if it should be
```suggestion
def ensure_bool(s: str | Any) -> bool:
```

or at least

```suggestion
def ensure_bool(s: str | bool | int) -> bool:
```
since we do test for `int` in test_assure_bool

Reply:
Done in 43cb9f03d.